stacks-templates/core.json

```diff
@@ -0,0 +1,52 @@
+{
+  "ordered": true,
+  "TAG": null,
+  "LABEL": "org.opencontainers.image.licenses=\"GPL-2.0-or-later\" \\\n      org.opencontainers.image.source=\"https://github.com/rocker-org/rocker-versioned2\" \\\n      org.opencontainers.image.vendor=\"Rocker Project\" \\\n      org.opencontainers.image.authors=\"Carl Boettiger <cboettig@ropensci.org>\"",
+  "stack": [
+    {
+      "IMAGE": "r-ver",
+      "FROM": null,
```

Comment:
Would it not be better to have the script automatically update the stack file, so these fields don't show up as `null`?  It seems less-than-ideal to me to have a config file that hides the actual values here

Reply:
This file is a template for `core-x.x.x.json` that is referenced from `make-stacks.R`.
The role of `make-stacks.R` is to read this template file and generate stack files with appropriate values for the `null` fields.
This means that if we want to modify the dockerfiles, we will need to modify two files: this template file and the script.
Currently it is placed in a newly created directory called `. /stacks-templates/`, but I will move it if there is a better location.

Reply:
I'm sorry if I misinterpreted your intent.
It is also a good idea to include a default value that points to the latest version, such as `latest` or `devel`, instead of `null`.
However, if we set `null` as default values, we may notice the anomaly when `docker build` fails if the variable is not assigned correctly by the script for some reason.

Reply:
yeah, whoops, didn't realize this was a separate stacks-template file.  I'd rather not duplicate all of our stacks again to have separate `stacks` and `stacks-templates`. I think the current script could effectively use the current stack as the 'template' and update/overwrite the previous fields, rather than have these separate templates with `null` entries?

 Ideally we'd consolidate these so we have only one JSON file per stack (obviously will require updating the `make-dockerfiles.R` file as well.  In that case, the stack file would presumably use a JSON array for the version list, and the `make-dockerfiles.R` would have to understand the logic to make the mutliple dockerfiles.

Reply:
Done.

As for the core stackfiles, we have devel, so I changed it to use that as the template.
For other types of stack files, we may need to include a process to dynamically select a template from the file name.